### DISCUSSION TITLE
fast scrolling may reveal momentary blank areas of unrendered content.

### DISCUSSION THREAD

**Discussion by NikhilMasurkar** [Shopify/flash-list#856] [General]
why blank area comes after scrolling fast 

https://github.com/Shopify/flash-list/assets/101415426/0de4dd3c-35c7-4d60-a5ee-a76e20b4f2a3

**Comment by parakhjain23** [comment]
Have you found any solution for this

**Reply by NikhilMasurkar** [reply]
not yet parakh

**Reply by joaogabrieldasilva** [reply]
any updates on this one?

**Reply by maXXCZ1** [reply]
Just to be sure. Are you testing this on real device with production build (not in development mode)? I got pretty heavy components that are rendering similarly, but in bundled production build it is quite different story.
